src/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { resolveApp } from './utils';
+
+export const paths = {
```

Comment:
I didn't see where `resolveApp` can throw, so I removed the `try-catch` statement